inc/classes/optimization/CSS/class-abstract-css-optimization.php

```diff
@@ -1,8 +1,8 @@
 <?php
```

Comment:
As we are touching this file, let's move and rename per our new standards/architecture.